group_vars/mqtt_servers.yml

```diff
@@ -0,0 +1,32 @@
+---
+albs_web_ip: 10.0.18.39
```

Comment:
I think this is fine for now but long term it should get wrapped into a larger ALBS role at which point we may want to rename the vars.